app/services/user.js

```diff
@@ -0,0 +1,27 @@
+const errors = require('../errors'),
+  { user: User } = require('../models'),
+  encryptationHelper = require('../helper/encryptation');
+
+const logger = require('../logger');
+
+exports.createUser = user =>
+  encryptationHelper
+    .encrypt(user.password)
+    .then(hash =>
+      User.createModel({
+        firstName: user.firstName,
+        lastName: user.lastName,
+        email: user.email,
+        password: hash
+      })
+        .then(newUser => {
+          logger.info('User created');
+          return newUser;
+        })
+        .catch(e => {
+          throw errors.databaseError(e.message);
+        })
+    )
+    .catch(e => {
+      logger.error(e.message);
```

Comment:
here you still need to rethrow the error so the error handler can pick it up

Reply:
Yes, I  forgot this one. I ve already change it